Terminal.Gui/Drivers/OutputBufferImpl.cs

```diff
@@ -291,6 +291,8 @@ private void WriteWideGrapheme (int col, int row, string grapheme)
             // Mark second column as replacement to indicate partial overlap
             if (col + 1 < Cols)
             {
+                // NOTE: This is dead code and apparently never called.
+                // NOTE: See AddStr_WideChar_FirstColumnOutsideClip_SecondColumnInside_CurrentBehavior
```

Comment:
Yes, you're right, because that piece of code will never be executed since it's already been cleared by ClearViewport and will only contain a space.